sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatServiceVersion.java

```diff
@@ -10,7 +10,7 @@
  */
 public enum ChatServiceVersion implements ServiceVersion {
     V2021_03_07("2021-03-07"),
```

Comment:
@LuChen-Microsoft  @juancamilor  Should we leave the stable version here or we should remove it?

Reply:
That would be a breaking change, so we have to keep it :)